### ISSUE TITLE
Removing Irrelevant Feature [GSSoC'23 ]

### ISSUE THREAD

**Issue by Rachit1210** [AbhiPatel10/AISKCON-CONSTRUCTION#283]
I want to remove these content displaying on each component as they can be navigated through navbar, also they are not looking good in the website.
In Services Page:-
<img width="962" alt="image" src="https://github.com/AbhiPatel10/AISKCON-CONSTRUCTION/assets/84730626/3662cb98-38c6-4836-908a-11e9d2caf88f">

In About Page:-
<img width="960" alt="image" src="https://github.com/AbhiPatel10/AISKCON-CONSTRUCTION/assets/84730626/d611be97-d887-4d23-af03-5ac5e027f1ab">

In Contact Page:-
<img width="960" alt="image" src="https://github.com/AbhiPatel10/AISKCON-CONSTRUCTION/assets/84730626/ed4e922f-53ed-47ca-ae47-3da4c686978d">

@kartikayasija Please Have a look and do suggest if anything else to be done to replace this.

**Comment by piyushlakheda** [comment]
Yes, @Rachit1210 you can remove those! And some other person is already assigned with the navbar active links issue. so you can leave that part

**Comment by Rachit1210** [comment]
Ok I will remove only the red circle parts
